packages/compartment-mapper/src/import-archive.js

```diff
@@ -308,7 +308,8 @@ export const parseArchive = async (
       __shimTransforms__,
       Compartment,
     });
-    return compartment.import(moduleSpecifier);
+    // eslint-disable-next-line dot-notation
+    return compartment['import'](moduleSpecifier);
```

Comment:
whats this change for?

Reply:
Manual evasive transform for the `import` censor.